src/env_cli_vars.nim

```diff
@@ -0,0 +1,139 @@
+    uri* {.
+      defaultValue: ""
+      desc: "status-app:// URI to open a chat or other"
+      name: "uri" .}: string
+    testMode* {.
+      defaultValue: false
+      desc: "Determines if the app should be run in test mode"
+      name: "TEST_MODE"
+      abbr: "test-mode" .}: bool
+    enableWallet* {.
+      defaultValue: true
+      desc: "Determines if the wallet section is enabled"
+      name: "ENABLE_WALLET"
+      abbr: "enable-wallet" .}: bool
+    openSeaApiKey* {.
+      defaultValue: ""
+      desc: "Sets open sea api key"
+      name: "OPENSEA_API_KEY"
+      abbr: "open-sea-api-key" .}: string
+    genacheNetworkRpcUrl* {.
+      defaultValue: ""
+      desc: "Sets ganache network rpc url"
+      name: "GANACHE_NETWORK_RPC_URL"
+      abbr: "ganache-network-rpc-url" .}: string
+    poktToken* {.
+      defaultValue: ""
+      desc: "Sets pokt token"
+      name: "POKT_TOKEN"
+      abbr: "pokt-token" .}: string
+    infuraToken* {.
+      defaultValue: ""
+      desc: "Sets infura token"
+      name: "INFURA_TOKEN"
+      abbr: "infura-token" .}: string
+    infuraTokenSecret* {.
+      defaultValue: ""
+      desc: "Sets infura token secret"
+      name: "INFURA_TOKEN_SECRET"
+      abbr: "infura-token-secret" .}: string
+    alchemyEthereumMainnetToken* {.
+      defaultValue: ""
+      desc: "Sets alchemy ethereum mainnet token"
+      name: "ALCHEMY_ETHEREUM_MAINNET_TOKEN"
+      abbr: "alchemy-ethereum-mainnet-token" .}: string
+    alchemyEthereumGoerliToken* {.
+      defaultValue: ""
+      desc: "Sets alchemy ethereum goerli token"
+      name: "ALCHEMY_ETHEREUM_GOERLI_TOKEN"
+      abbr: "alchemy-ethereum-goerli-token" .}: string
+    alchemyArbitrumMainnetToken* {.
+      defaultValue: ""
+      desc: "Sets alchemy arbitrum mainnet token"
+      name: "ALCHEMY_ARBITRUM_MAINNET_TOKEN"
+      abbr: "alchemy-arbitrum-mainnet-token" .}: string
+    alchemyArbitrumGoerliToken* {.
+      defaultValue: ""
+      desc: "Sets alchemy arbitrum goerli token"
+      name: "ALCHEMY_ARBITRUM_GOERLI_TOKEN"
+      abbr: "alchemy-arbitrum-goerli-token" .}: string
+    alchemyOptimismMainnetToken* {.
+      defaultValue: ""
+      desc: "Sets alchemy optimism mainnet token"
+      name: "ALCHEMY_OPTIMISM_MAINNET_TOKEN"
+      abbr: "alchemy-optimism-mainnet-token" .}: string
+    alchemyOptimismGoerliToken* {.
+      defaultValue: ""
+      desc: "Sets alchemy optimism goerli token"
+      name: "ALCHEMY_OPTIMISM_GOERLI_TOKEN"
+      abbr: "alchemy-optimism-goerli-token" .}: string
+    defaultTorentConfigPort* {.
+      defaultValue: 0
+      desc: "Sets default torrent config port"
+      name: "DEFAULT_TORRENT_CONFIG_PORT"
+      abbr: "default-torrent-config-port" .}: int
+    defaultWakuV2Port* {.
+      defaultValue: 0
+      desc: "Sets default waku v2 port"
+      name: "DEFAULT_WAKU_V2_PORT"
+      abbr: "default-waku-v2-port" .}: int
+    statusPort* {.
+      defaultValue: 0
+      desc: "Sets Waku V1 config port"
```

Comment:
@richard-ramos do we still need the Waku v1 port? AFAIK we don't, it should be removed from the runtime config.

Reply:
It is still possible to use WakuV1 by switching fleets. I'd keep the option until WakuV1 support is removed from status-go